### PR TITLE
Abdulmajeed98/issue9

### LINKED ISSUE #9: create Contact Us section

### PR DESCRIPTION
**Pull request by Abdulmajeed98**
@Revan99 

Take a look at the _**Contact Us **_ section.

fix #9

### CODE REVIEW

File: src/components/contact/ContactUs.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    { id: 3, icon: <FaMapMarkerAlt className='w-full h-full' />, title: 'Address', content: 'Holy Zard St, Erbil, Iraq, 44001', href: '#' },
+]
+
+const ContactUs = () => {
+    // const [formData, setFormData] = useState({
+    //     name: '',
+    //     email: '',
+    //     phoneNumber: '',
+    //     message: ''
+    // })
```
[the remark this answers is not in the document]
Comment (Abdulmajeed98) [reply src/components/contact/ContactUs.jsx:26]: removed

File: src/components/contact/ContactInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const ContactInfo = ({ icon, title, content, href }) => {
+    console.log(href);
```
[the remark this answers is not in the document]
Comment (Abdulmajeed98) [reply src/components/contact/ContactInfo.jsx:2]: removed

File: src/components/contact/ContactInfo.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    return (
+        <a href={href} className="flex-1 flex items-center justify-center" style={{ textDecoration: 'none' }}>
+            <span className={`flex-1 inline-block w-10 h-10 text-xl ${title === 'Phone' && 'animate-wiggle'} ${title === 'Email' && 'animate-pulse'} ${title === 'Address' && 'animate-bouncing'}`}>{icon}</span>
```
[the remark this answers is not in the document]
Comment (Abdulmajeed98) [reply src/components/contact/ContactInfo.jsx:5]: just moved the animation values into the object in the **_ContactUs.jsx_** check it out now

**Comment by Abdulmajeed98** [comment]
@Revan99 
the changes is done, you can take a look now

**Review by Revan99 — APPROVED** [review]

**Pull request merged**